src/modules/baseCompare/baseCompare.test.js

```diff
@@ -0,0 +1,80 @@
+import { useDatabase } from 'server/test/utils'
+import factory from 'server/test/factory'
+import { getBaseScreenshotBucket } from './baseCompare'
+
+describe('baseCompare', () => {
+  useDatabase()
+
+  describe('getBaseScreenshotBucket', () => {
+    it('should work with a rebased PR', async () => {
+      // History:
+      // * | - 3304d38afd1838cadf4e859de46b495fb347efec
+      // * 0e4c8e8ee37c9f62c4e77f5b8d1abe3ebe845a92
+      const repository = await factory.create('Repository')
+      const screenshotBucket = await factory.create('ScreenshotBucket', {
+        name: 'test-bucket',
+        branch: 'test-branch',
+        commit: '0e4c8e8ee37c9f62c4e77f5b8d1abe3ebe845a92',
+        repositoryId: repository.id,
+      })
+
+      const baseScreenshotBucket = await getBaseScreenshotBucket({
+        owner: 'callemall',
+        repo: 'material-ui',
+        baseCommit: '0e4c8e8ee37c9f62c4e77f5b8d1abe3ebe845a92',
+        compareCommit: '3304d38afd1838cadf4e859de46b495fb347efec',
+        repositoryId: repository.id,
```

Comment:
I'm gonna refactor to use a `repository` as input and integrate the block with the system.